backend/models/test.model.ts

```diff
@@ -51,7 +51,8 @@ export interface TextMetadata {
  * This interface contains additional information about an image component
  */
 export interface ImageMetadata {
-  src: string;
+  url: string;
+  filePath: string;
```

Comment:
why do we need both `url` and `filepath`? sure you have your reasons, but curious!

Reply:
`url` is the "preview link" we can use to display the photo but has expiration time
`filePath` persists after expiration and is used to access the photo and generate new url

Reply:
actually with the way it currently works, we don't really need to store the url cause we regenerate it every time

Reply:
do you think makes more sense to just store the `filePath`? and then return only the `url`

Reply:
i think that makes more sense to me! but if it's a lot of extra effort it's probably fine as is

Reply:
i think i'm gonna leave as is actually bc it gets a bit ugly converting between the two types - we can refactor later if needed